README.md

```diff
@@ -68,14 +68,14 @@ Booting into read-only snapshots is fully supported when choosing btrfs as the f
   * (only when using the grub-btrfsd daemon)[inotify-tools](https://archlinux.org/packages/community/x86_64/inotify-tools/)
 
 - - -
-### 📚 Usage
-After installation the grub main menu needs to be generated to make a menuentry for the snapshots sub-menu. Depending on the Linux distribution the commands for that are different:
+### 📚 Manual usage of grub-btrfs
+To manually (re)generate grub snapshot entries you can run `sudo /etc/grub.d/41_snapshots-btrfs` which updates `grub-btrfs.cfg`. You then need to re-install GRUB by running one of the following commands:
```

Comment:
"re-install" is the wrong wording here. `grub-mkconfig` only generates the menu. Reinstalling grub could frighten people.